src/classes/CRLP_RollupCMT.cls

```diff
@@ -299,7 +299,7 @@ public class CRLP_RollupCMT {
                 this.summaryObjectLabel = rlp.Summary_Object__r.Label;
                 this.summaryFieldLabel = rlp.Summary_Field__r.Label;
                 this.detailObjectLabel = rlp.Detail_Object__r.Label;
-                this.detailFieldLabel = UTIL_Describe.getFieldLabel(rlp.Detail_Object__r.QualifiedApiName, rlp.Detail_Field__r.QualifiedApiName);
+                this.detailFieldLabel = rlp.Detail_Field__r.QualifiedApiName == null ? null : UTIL_Describe.getFieldLabel(rlp.Detail_Object__r.QualifiedApiName, rlp.Detail_Field__r.QualifiedApiName);
```

Comment:
Does this change also need to be made in the filterGroup code?

Reply:
Also, I recommend changing this to read: `rlp.Detail_Field__c == null` so that you're checking the root field value versus the relationship of QualifiedFieldName against a potentially null `Detail_Field__c` value.

Reply:
@randi274 no because it'll never be null there, right? all fields are required.